veggieseasons/lib/data/local_veggie_provider.dart

```diff
@@ -2,591 +2,281 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/cupertino.dart';
 import 'package:veggieseasons/data/veggie.dart';
 
 class LocalVeggieProvider {
   static List<Veggie> veggies = [
     Veggie(
       id: 1,
       name: 'Apples',
-      imageAssetPath: 'assets/images/placeholder.png',
+      imageAssetPath: 'assets/images/apple.jpg',
       category: VeggieCategory.fruit,
       shortDescription: 'Green or red, they\'re generally round and tasty.',
-      accentColor: 0xff206010,
+      accentColor: Color(0x40de8c66),
       seasons: [Season.winter, Season.spring, Season.summer, Season.autumn],
```

Comment:
You're talking about having something more granular than seasons here, right?

When I went looking for data on harvest times, it was always in season format rather than something more specific like months. I'm sure I could find something better if I took the time to really dig in, especially now that we have a final list of vegetables.

We were at one point talking about using geo to determine whether a veggie was in season, though, resulting in questions like "When do strawberries ripen in Australia?" That would definitely require something more sophisticated. 

I'm going to leave this as-is for now, and think about how far I can go with the accuracy here. Feel free to grab me if you want to talk it over.